scripts/telegraf.service

```diff
@@ -8,7 +8,8 @@ EnvironmentFile=-/etc/default/telegraf
 User=telegraf
 ExecStart=/usr/bin/telegraf -config /etc/telegraf/telegraf.conf -config-directory /etc/telegraf/telegraf.d ${TELEGRAF_OPTS}
 ExecReload=/bin/kill -HUP $MAINPID
-Restart=on-failure
+Restart=always
+RestartForceExitStatus=SIGPIPE
```

Comment:
Fixed, sorry , got confused doing merge conflict while multi tasking . thanks for catching this up!